apps/catalogue/src/network/NetworksHome.vue

```diff
@@ -1,7 +1,6 @@
 <template>
   <div class="container-fluid">
-    {{ searchFilter }}
-    <h1>Welcome to the European Cohort Network Catalogue.</h1>
+    <h1>Welcome to the European Networks Health Data and Cohort Catalogue.</h1>
```

Comment:
Suggest to remove the 'Welcome to the' from the header and move to <p> , as its not really a title, gets a bit tiring after the first visit ( and does not help with seo i would guess )

Reply:
good suggestion

Reply:
lol github renders my ```<p>``` as a paragraph